api/v1beta2/awsmachine_webhook.go

```diff
@@ -192,10 +192,6 @@ func (r *AWSMachine) validateRootVolume() field.ErrorList {
 		}
 	}
 
-	if r.Spec.RootVolume.DeviceName != "" {
```

Comment:
How about we log this info and not return error?

Reply:
Good idea!  Updated :)